### PR TITLE
Prepare version 0.11

### PR DESCRIPTION
**Pull request by tlogan2000**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,12 +1,16 @@
 Changes
 *******
 
-TBD (unreleased)
+0.11.0 (2023-06-13)
 ================
```
Comment (Zeitsperre) [inline CHANGES.rst:5]:
```suggestion
===================
```

Comment (Zeitsperre) [reply CHANGES.rst:5]: For reference, the underline needs to match the length of the line; Sphinx will complain otherwise.

File: CHANGES.rst [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+* Added xclim yml module support
+* Added humidex days above calculation via yml module
+* Reimplmented streamflow indicators via yml module (adjust for xclim 0.41 breaking changes)
+* Fixed bug for CanDSC-U6 ensemble "26models" list
 * Fixed iter_local when depth > 0 to avoid all files to be considered twice
 * Revised documentation configuration on ReadTheDocs to leverage Anaconda (Mambaforge)
 * Minor adjustments to dependency configurations
-* Update to xclim 0.40.0.
```
Comment (Zeitsperre) [inline CHANGES.rst:9]: Not moving up to 0.43.0 yet?

**Review by Zeitsperre — APPROVED** [review]

File: CHANGES.rst (hunk 1 above)
Comment (tlogan2000) [reply CHANGES.rst]: Ok changed

File: CHANGES.rst (hunk 2 above)
Comment (tlogan2000) [reply CHANGES.rst:9]: My mistake... uppate is to 0.43

**Pull request merged**